testing/jenkins_remote_access.py

```diff
@@ -144,16 +132,6 @@ def remove_slave_info(labelString):
     )
 
 
-def remove_all_jobs(regex):
```

Comment:
Well i wouldn't delete this because this is handy if you want to delete all jobs of a jenkins instance from a developer machine. Found myself deleting jobs manually when i want to restart jenkins and jobs have cron schedulers that prevent jenkins from restarting.

Reply:
I do all this stuff from a developer machine using `test.sh`